commands/system/vpn/vpn-config.template.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This is a template which needs further setup. Duplicate the file,
+# remove `.template.` from the filename, and set the name of
+# your VPN config from System Preferences.
+
+# Name of your VPN Config
+export VPN_NAME="VPN (IKEv2)"
+
+if [ -z "$VPN_NAME" ]; then
+  echo "\$VPN_NAME is empty";
```

Comment:
Will make sense here to indicate in which file the user should configure the VPN name.

```suggestion
  echo "\$VPN_NAME is empty";
  echo "Please, rename it in \"vpn-config.sh\".";
```